managed-identity-keyvault/src/main/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+# Specify the URI of your Key Vault (e.g.: https://name.vault.azure.net/)
+azure.keyvault.uri=https://<your-keyvault-name>.vault.azure.net/
```

Comment:
Hi,
I believe you forget to mention changing this before mvn clean package in readme.md.

Reply:
+1